src/components/molecules/CurrencyMaturityDropdown/CurrencyMaturityDropdown.tsx

```diff
@@ -100,30 +100,32 @@ export const CurrencyMaturityDropdown = ({
         (options: any[]) => {
             const { column, direction } = sortState;
 
-            if (!column) {
-                return options;
-            }
-
             return options.sort((a, b) => {
-                let aValue, bValue;
-
-                if (column === 'apr') {
-                    aValue = parseFloat(a.apr.replace('%', ''));
-                    bValue = parseFloat(b.apr.replace('%', ''));
-                } else if (column === 'maturity') {
-                    aValue = a.maturity;
-                    bValue = b.maturity;
-                } else {
-                    aValue = a[column];
-                    bValue = b[column];
-                }
+                if (column) {
+                    let aValue, bValue;
+
+                    if (column === 'apr') {
+                        aValue = parseFloat(a.apr.replace('%', '')) || 0;
```

Comment:
is there a chance a.apr.replace('%', '') is null or undefined?

Reply:
I am adding a conditional "?" to handle undefined, otherwise aValue (and bValue) will be safe with 0 @as472